### DISCUSSION TITLE
Can't install: attempt to automatically shut down Windows Explorer fails

### DISCUSSION THREAD

**Discussion by SEE-ICT** [WinMerge/winmerge#1368] [Q&A]
When I attempt to install, I get a message that Windows Explorer needs to be shut down. When I select the option calling for the installer to do the shutdown itself, I get "Setup was unable to automatically close all applications". What can I do to solve this? Thanks!

P.S. To be clear, the installer is running as Administrator.

**Comment by sdottaka** [comment]
During installation, you will be asked to exit Windows Explorer if the ShellExtension for WinMerge is loaded into Explorer.
Normally, Explorer will exit and the installation will continue, but for some reason it may not work.
I have encountered this problem several times and have yet to find a good solution.

Are you using WinMerge with multiple users logged on to the same machine at the same time?

If not, a simple solution is to restart the computer and run the WinMerge installer before bringing up the Explorer context menu.

If you show the Explorer context menu before running the installer, WinMerge's ShellExtension will be loaded into Explorer and the same problem will occur.

**Comment by SEE-ICT** [comment]
Only one user logged on at a time. Maybe I should uninstall the earlier WinMerge version and then try installing the new? I'll give that a shot and if it doesn't work, I'll try your restart solution. Thanks for the suggestion!

**Comment by SEE-ICT** [comment]
Well, I uninstalled the previous version of WinMerge, rebooted and reinstalled the new version. Which worked this time. Whether it was the uninstall or the reboot that did it, I don't know. But whatever, I'm now in good shape. Thanks for the quick reply! It certainly got me looking in the right direction.
